docs/CLI-COMMANDER-PATTERN.md

```
@@ -0,0 +1,131 @@
+const program = new Command();
+program.addCommand(new ListTasksCommand());
+```
+
+### In Existing Scripts
+```javascript
+// Gradual adoption
+const listCmd = new ListTasksCommand();
+program.addCommand(listCmd);
+```
+
+### Programmatic Usage
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)